notebooks/contributing/formatting.md

```diff
@@ -0,0 +1,289 @@
+
+
+## Notebook Introduction
+
+Every tutorial notebook requires an introduction section. The Markdown cells required for the introduction are described below.
+
+### Cell 1: Title Cell
+
+The first cell contains the title and the author for the tutorial. It is formatted as follows.
+The title is in quotations. You will need to add yourself as an author in the `myst.yml` file in the root directory of the repository.
+
+```
+---
+title: "My Notebook Title In Quotes"
+authors:
+  - id: devincowan
+---
+```
+
+### Cell 2: Tutorial Difficulty
+
+Here, we indicate whether the notebook contains introductory, intermediate or advanced content. The author is required to choose
+from one of the following [MyST admonitions](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html), which is rendered when the webpage is built.
+
+```
+:::{admonition} Introductory notebook
+:class: hint
+This tutorial teaches basic functionality within SimPEG and is a good
+entry point for new users.
+:::
+```
+
+```
+:::{admonition} Intermediate notebook
+:class: caution
+This tutorial focusses on intermediate level functionality within SimPEG.
+Basic functionality within SimPEG is not discussed in detail, as we assume
+the user is already familiar.
+:::
+```
+
+```
+:::{admonition} Advanced notebook
+:class: danger
+This tutorial focusses on advanced functionality within SimPEG. Basic and
+intermediate level functionality are not discussed in detail, as we assume
+the user is already an experienced SimPEG user.
+:::
+```
+
+### Cell 3: Computational Requirements
+
+Here, we specify the computational requirements to run the notebook. Whereas some notebooks can be run easily on laptop computers, some notebook may need to be exported as Python files and run using cluster computing. The author is required to choose from one of the following [MyST admonitions](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html), which is rendered when the webpage is built.
+
+```
+:::{admonition} Light-weight notebook
+:class: hint
+This tutorial requires minimal computational resources and can be executed
+quickly in the background while other computer processes are running.
+:::
+```
+
+```
+:::{admonition} Medium-weight notebook
+:class: caution
+Requires moderate computational resources. Run-times may exceed several
+minutes and require up to 8 GB of available RAM.
+:::
+```
+
+```
+:::{admonition} Computationally intensive notebook
+:class: danger
+The computational resources required to execute this notebook exceed those
+provided by standard laptop computers. To execute the notebook, please deploy
+to a cluster computing environment.
+:::
+```
+
+### Cell 4: Keywords, Summary and Learning Objectives
+
+In this cell, the contributor must provide a set of relevant keywords. E.g.
+
+```
+**Keywords:** gravity inversion, sparse-norm inversion, integral formulation, tree mesh.
```

Comment:
Just sharing an idea, not to tackle it now: MystMD has support for specifying [keywords through the frontmatter](https://mystmd.org/guide/frontmatter#available-frontmatter-fields). I think at some point it would be better to use those instead of hard-coding the keywords as Markdown text.